diagnostic.md

```diff
@@ -31,39 +34,44 @@ the `form` DOM node. (Assume the list item has a class of "listerItem").
 Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
-<!-- your answer starts here -->
+<!-- $("li").show -->
+
+<!-- $("li").()
+
+Setters
+$(.target).text("Hola")
+$(.target).var-->
 
-<!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+<!-- It is a tree version of the HTML file, it allows us to access and modify. -->
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.  A web browser implements this interface and renders it for you to view and interact with.